components/automate-ui/src/app/pages/project/list/project-list.component.ts

```diff
@@ -286,4 +277,24 @@ export class ProjectListComponent implements OnInit, OnDestroy {
     // TODO: check how often this is hit
     return result;
   }
+
+  public getButtonText(): string {
+    if (this.applyRulesInProgress) {
+      return `Updating Projects ${Math.round(this.percentageComplete * 100)}%...`;
+    }
+    if (this.projectsHaveStagedChanges
+      || this.updateProjectsCancelled
+      || this.updateProjectsFailed) {
+      return 'Update Projects';
+    }
+    return 'Projects Up-to-Date';
```

Comment:
`Up-To-Date` was intentional from @susanev. Should change back or chat with her.

Reply:
there is was just a bunch of miscommunication, originally it was designed as 'Up-to-date`, but then @blakestier suggested `Up-to-Date` which i think is the way to go. but it ened up getting changed to `Up-To-Date`.. so all of that is to say i support the change here by @msorens